nipype/exceptions.py

```diff
@@ -2,6 +2,10 @@ class NipypeError(Exception):
     pass
 
 
+class EngineError(Exception):
+    pass
+
+
```

Comment:
sure, I made this change because the `EngineError` was not implemented and it was giving me an error. Didn't have anything in mind about hierarchy, just wanted to remove the error.